README.md

```diff
@@ -51,6 +50,55 @@ See [Django Docs on the subject](https://docs.djangoproject.com/en/dev/ref/contr
         url(r'^admin/', include(massadmin.urls), kwargs={'admin_site': admin_site}),
         ```
 
+## Settings
+
+### Enable Mass Edit for specific models
+
+By default, all models registered in the admin will get `Mass Edit` action.
+
+If you wish to disable this, add this to settings file:ž
```

Comment:
Here is probably forgotten character `ž`

Reply:
... and of course you know where on my keyboard this letter is located 😁